configs/11.0/build.mk

```diff
@@ -1 +0,0 @@
-../10.0/build.mk
```

Comment:
This also breaks the -cross build.  I'll work on all of these issues and repush when ready.